packages/smooth_app/lib/pages/product/summary_card.dart

```diff
@@ -102,47 +102,55 @@ class _SummaryCardState extends State<SummaryCard> {
 
   Widget _buildLimitedSizeSummaryCard(double parentHeight) {
     totalPrintableRows = parentHeight ~/ SUMMARY_CARD_ROW_HEIGHT;
-    return Stack(
-      children: <Widget>[
-        ClipRRect(
-          borderRadius: ROUNDED_BORDER_RADIUS,
-          child: OverflowBox(
-            alignment: AlignmentDirectional.topStart,
-            minHeight: parentHeight,
-            maxHeight: double.infinity,
-            child: buildProductSmoothCard(
-              header: _buildProductCompatibilityHeader(context),
-              body: Padding(
-                padding: SMOOTH_CARD_PADDING,
-                child: _buildSummaryCardContent(context),
+    return Padding(
+      padding: const EdgeInsets.only(
+        right: SMALL_SPACE,
+        left: SMALL_SPACE,
+        top: VERY_SMALL_SPACE,
+        bottom: VERY_SMALL_SPACE,
+      ),
```

Comment:
Why not use a EdgeInsets.symmetric ?

Reply:
Sure, we can do that. I just looked at the margin for "Welcome to Open Food Facts" card which was the SmoothCard defined in smooth_card.dart. It didn't use symmetric so I just tried to stick with how it was done earlier.

<img width="611" alt="Screen Shot 2022-04-14 at 9 38 40 AM" src="https://user-images.githubusercontent.com/49199047/163434470-56e0e21a-37c4-4a11-91e9-75f4b33bd8d9.png">

Reply:
Could you also change it here, please?

Reply:
Done, have deleted two iOS related files which somehow got pushed am not sure how.

Reply:
There was some mess. Had pushed Info.plist and project.pbxproj ios files by accident. I believe these files got changed because I had to change the Team and Bundle Identifier since my apple id isn't linked with your team and I was required to do that in order to run the app on my iPhone. Anyway, have pushed the latest versions of the files and I guess its all sorted but I believe these files need to be in .gitignore. Should I open an issue for that ?

Reply:
Sure @Laytoder I don't know if we need to have them commited or not but checking doesn't hurt